modules/json_content/src/Form/UserLoginForm.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\json_content\Form;
+
+use Drupal\user\Form\UserLoginForm as UserLoginFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+class UserLoginForm extends UserLoginFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    $account = $this->userStorage->load($form_state->get('uid'));
```

Comment:
The parent sets login destinations which we want to override. So I don't think so.